.eslintrc.json

```diff
@@ -7,7 +7,8 @@
   ],
   "parser": "babel-eslint",
   "env": {
-    "browser": true
+    "browser": true,
+    "jest": true
```

Comment:
just curious, what does it do?

Reply:
It adds `describe`, `it`, `expect`, etc. as global variables. This prevents the lint from going crazy in the tests. The tests might not be linted, but my editor certainly doesn't care :)

Reply:
👍